tests/metrics/storage/blogbench_dockerfile/Dockerfile

```diff
@@ -15,7 +15,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 # URL for blogbench test and blogbench version
 ENV BLOGBENCH_URL "https://download.pureftpd.org/pub/blogbench"
-ENV BLOGBENCH_VERSION 1.1
+ENV BLOGBENCH_VERSION 1.2
```

Comment:
@GabyCT @dborquez Does the `DOCKERFILE_VERSION` need to be updated as well?
I dont see it being used anywhere, so wanted to know if it needs an update as well.

Reply:
@amshinde no you do not need it